client/reader/following-manage/sitesWindowScroller.jsx

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Nice one. Can we also rename this file as per https://github.com/Automattic/wp-calypso/pull/13060#issuecomment-293917394?